common/app/routes/Challenges/views/quiz/Quiz.jsx

```diff
@@ -140,7 +140,7 @@ export class QuizChallenge extends PureComponent {
     const currentIndex = this.props.currentIndex;
     const question = this.props.description[currentIndex];
     return (
-      <div>
+      <div key={currentIndex.toString()} >
```

Comment:
We should avoid using an index as a key where possible, as they are only unique within the context of the current array. We could use the challenge `dashedName` Appended with the current index. `dashedName` should be available under `this.props.meta`

To expand on the index thing. Say we have the first question rendered, then the user changes challenge, react would see that the keys haven't changed so it could skip the render of the new question.